data/settings.py

```diff
@@ -10,7 +10,7 @@
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.getenv("DEBUG", False))
+DEBUG = True if (os.getenv('DEBUG') == 'true') else False
```

Comment:
You may want to double check with @nusnewob what value he is using in the Docker image.